modules/core/src/main/java/org/apache/ignite/internal/processors/cache/transactions/IgniteTxManager.java

```diff
@@ -180,6 +184,22 @@
             DFLT_LONG_OPERATIONS_DUMP_TIMEOUT
     );
 
+    /** Metric name for total system time on node. */
+    public static final String METRIC_TOTAL_SYSTEM_TIME = "totalNodeSystemTime";
+
```

Comment:
@agoncharuk Looks like we are implementing metrics here, but I thought this issue is about binary protocol. It is a bad practice to solve several problems in one issue. Lets move this changes to separate issue?